instance/autobuy.go

```diff
@@ -38,7 +38,7 @@ func (in *Instance) abFishingPole(_ discord.Message) {
 	if trigger == nil || trigger.Value != fishCmdValue {
 		return
 	}
-	in.sdlr.ResumeWithCommand(&scheduler.Command{
+	in.sdlr.PrioritySchedule(&scheduler.Command{
```

Comment:
And here..

Reply:
I'll explain, when I was testing stuff, resumewithcommand glitched. Basically it would execute pls search then it would execute pls buy rifle 1 which wouldnt be executed as the bot will say "not a valid search option" thus this change was required